chapters/introduction.tex

```diff
@@ -2,32 +2,20 @@ \chapter{Introduction}\label{introduction1}
 
 \section{Overview of Modelica}\label{overview-of-modelica}
 
-Modelica is a language for modeling of physical systems, designed to
-support effective library development and model exchange. It is a modern
-language built on acausal modeling with mathematical equations and
-object-oriented constructs to facilitate reuse of modeling knowledge.
+Modelica is a language for modeling of cyber-physical systems, supporting acausal modeling with mathematical equations and object-oriented constructs to facilitate modeling from first principles and reuse of modeling knowledge.
```

Comment:
It's all about _modeling_ then. 😄

Reply:
Right.  Synonyms, anyone?

Reply:
Is this better?
```suggestion
Modelica is a language for modeling of cyber-physical systems, supporting acausal connection of components governed by mathematical equations to facilitate modeling from first principles.
It provides object-oriented constructs that facilitate reuse of models, and can be used conveniently for modeling complex systems containing, e.g., mechanical, electrical, electronic, magnetic, hydraulic, thermal, control, electric power or process-oriented subcomponents.
```

Reply:
I prefer the first variant, since the latter:

- Introduces "connection" which is begins to get into internals that we should avoid in an overview.
- It only states that the connections are acausal, not the equations themselves. This is an important distinction since there have been competing technologies that don't fully embrace acausal equations, but sort of allow acausal connections.

Reply:
> I prefer the first variant, since the latter:
> - Introduces "connection" which is begins to get into internals that we should avoid in an overview.

I didn't intend _connection_ to be understood in the technical sense of Modelica, but in a generic and harmless sense that I don't think hurts in an introduction like this one.

> - It only states that the connections are acausal, not the equations themselves. This is an important distinction since there have been competing technologies that don't fully embrace acausal equations, but sort of allow acausal connections.

Can't we consider acausality a property of mathematical equations that should be naturally understood by our readers?

I believe the best we can do at the moment is to try to come up with some alternatives to be further discussed at the phone meeting on Wednesday, so let's just consider the variant above as one proposal with more moderate use of _modeling_ compared to the formulation it intends to replace.  Maybe you could provide some other proposal for how to avoiding all the _modeling_?

Reply:
@gwr69: I like to add that we address cyber-physical systems very well now in the Modelica Association with the new eFMI Standard developed in the ITEA3 project EMPHYSIS (cf. https://emphysis.github.io/).

And of course, eFMI is tested with Modelica models as basis (cf. https://github.com/modelica/efmi-testcases). And industrial demonstrators (like an Audi A8 whose active dampening is controlled by a Modelica model deployed on the car's ECU, cf. https://www.mdpi.com/2076-0825/10/11/301).

So I would say it is not just a claim that Modelica supports cyber-physical systems -- we do successfully.

Reply:
Agreement to Suggested change above (in this thread)+restore trademark.